Apps/Playground/Scripts/config.json

```diff
@@ -245,23 +245,23 @@
         {
             "title": "Scissor test",
             "renderCount": 10,
-            "playgroundId": "#W7E7CF#29",
+            "playgroundId": "#W7E7CF#12",
             "referenceImage": "scissor-test.png",
-            "comments": "Reset playgroundId to #W7E7CF#12 when d3d12 mip-map issue is fixed"
```

Comment:
A comment is probably still useful here. I don't expect D3D12 to not have the support forever.